setup.py

```diff
@@ -1,6 +1,11 @@
 #!/usr/bin/env python
+import os
 from distutils.core import setup
 
+datadir = os.path.join('Schema')
```

Comment:
I think you have to explicitly find the path to the current file. I'm guessing that you've always been running the setup script whilst pwd is the folder it is in?

Reply:
This is what we used in our Django projects:

    PROJECT_DIRECTORY = os.path.join(os.path.abspath(os.path.dirname(__file__)),"../..")